app/controllers/index.js

```diff
@@ -1,23 +1,30 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    _ = require('underscore');
+const mongoose = require('mongoose'),
+  async = require('async'),
```

Comment:
Fix this too

Reply:
all these errors have been there. all I did on those files were to change ```var``` to ```const```

Reply:
Since you changed the file, you might want to just fix them. It is not healthy to include warnings in our code. The team has been fixing many hound issues since last week. It is normal to just fix this one too. Please bro